feat: Add a command to generate JSON Schema (#213)
megaease/easeprobe: docs/Manual.md

  # check the following link to see the time zone list
  # https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
  timezone: "America/New_York" #  default: UTC
```
# 8. Tools

## 8.1 EaseProbe JSON Schema

> We have a schema file can be used to validate the EaseProbe configuration file. It is available at [resources/schema.json](https://raw.githubusercontent.com/megaease/easeprobe/main/resources/schema.json).

Reviewer: ```suggestion
We have a JSON schema that can be used to validate your EaseProbe configuration. The schema can be found at [resources/schema.json](https://raw.githubusercontent.com/megaease/easeprobe/main/resources/schema.json).
```

> This schema file can be generated by the following command:

Reviewer: ```suggestion
The schema file can be generated at any time running the following command:
```

```bash
$ easeprobe -j > schema.json
```

> For VSCode, you can install the [YAML extension](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) and add the following configuration to your `settings.json` file:

Reviewer: ```suggestion
In order to use the schema with VSCode for validating your configuration, you need to install the [YAML extension](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) and add the following configuration to your `settings.json` file:
```

## 8.1 EaseProbe JSON Schema

We have a JSON schema that can be used to validate your EaseProbe configuration. The schema can be found at [resources/schema.json](https://raw.githubusercontent.com/megaease/easeprobe/main/resources/schema.json).

> The schema file can be generated at any time running the following command:

Reviewer: ```suggestion
The schema file can be generated at any time by running the following command:
```
